packages/catalog-generator/pom.xml

```diff
@@ -37,6 +37,11 @@
       <artifactId>camel-catalog</artifactId>
       <version>${version.camel}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-yaml-dsl</artifactId>
+      <version>${version.camel}</version>
+    </dependency>
```

Comment:
I have seen no changes of packages in the runtime code.
Shouldn't it be a test dependency? (or there was bug before? or I have missed the changes in runtime code requiring this dependency?)

Reply:
> Shouldn't it be a test dependency?

This is being used in tests, so yeah, if that's possible, then it would be better I think, how can be done?

> or there was bug before?

No, it wasn't